examples/host-function-library/plugin/plugin.go

```diff
@@ -0,0 +1,43 @@
+//go:build tinygo.wasm
```

Comment:
I suggest you also provide an easy way to compile the example by using something like `//go:generate tinygo build -o plugin/plugin.wasm -scheduler=none -target=wasi --no-debug plugin/plugin.go`

Reply:
Np. I did that here to generate the wasm extension to test my code https://github.com/jcchavezs/coraza-exec-action-wasm/blob/main/plugin_test.go#L1

Reply:
cool I guess this is the only thing before merge. nicely done!

Reply:
Oh, yes, sure. Added for this example and will fix it for another examples later.
Also, I saw that @jcchavezs is using `mage` in example, also like it this approach :+1:

Reply:
Yeah mage is great. Makefiles are painful for me as I was learning it again every time.